ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/MessageScrubber.java

```diff
@@ -87,6 +91,39 @@
       .put("/submit/sslreports/", "1585144") //
       .build();
 
+  private static final String REDACTED_SEARCH_CODE_VALUE = "other.scrubbed";
+
+  // The key format is <provider>.in-content:[sap|sap-follow-on|organic]:[<code>|none]
+  private static final Pattern DESKTOP_SEARCH_COUNTS_PATTERN = Pattern
+      .compile("(.+\\.in-content.[^:]+:)(.*)");
+
+  // The key format is <provider>:[tagged|tagged-follow-on|organic]:[<code>|none]
+  private static final Pattern DESKTOP_SEARCH_CONTENT_PATTERN = Pattern
+      .compile("([^:]+:[^:]+:)(.*)");
```

Comment:
I've switched to using two different regex expressions as the basis for matching and for parsing out which part is the `code` to be redacted. This should be more robust than the previous method of just stripping the part after the final `:` character.